mne/epochs.py

```diff
@@ -3174,6 +3175,10 @@ class Epochs(BaseEpochs):
 
     Notes
     -----
+    When Epochs are constructed using only a raw object (from the annotations
+    stored in the raw object), the duration of the events are ignored since
+    Epochs must be the same time length by design.
```

Comment:
I think this note should go into a `.. note::` section for the `raw` parameter – whose docstring should be updated anyway to suggest that annotations will be used if no events are being passed

Reply:
Also, this note is also confusing, because it mixes annotations and events ("the duration of the events are ignored").

Reply:
I think this comment wasn't addressed so far :)